sql/driver/tx.go

```diff
@@ -17,16 +17,39 @@
 
 package driver
 
-import "fmt"
+import "database/sql/driver"
 
 type tx struct {
 	conn *conn
 }
 
 func (t *tx) Commit() error {
-	return fmt.Errorf("TODO(pmattis): unimplemented")
+	_, err := t.Exec("COMMIT TRANSACTION")
+	return err
 }
 
 func (t *tx) Rollback() error {
-	return fmt.Errorf("TODO(pmattis): unimplemented")
+	_, err := t.Exec("ROLLBACK TRANSACTION")
+	return err
+}
+
+func (t *tx) Exec(query string, args ...interface{}) (driver.Result, error) {
```

Comment:
Is this method ever getting called? The `database/sql/driver.Tx` interface doesn't specify this signature. `database/sql` already holds onto a conn for the lifetime of the transaction.